iset.mm

```diff
@@ -120972,6 +121173,17 @@ C_ if ( A. k e. suc J ( Q ` ( i e. _om
       IEGEHZIJZKUJLZUFMZNZODEGUHUNUCUJJZUDUKUGUMKUCUJIPUOUEULUFUCUJQRSTUAUB $.
   $}
 
+  ${
+    $d x y $.
+    $( The Schroeder-Bernstein Theorem is equivalent to excluded middle.  This
+       is Metamath 100 proof #25.  (Contributed by Jim Kingdon,
+       13-Aug-2022.) $)
```

Comment:
This is confusing: Metamath 100 proof #25 is not about the equivalence of the Schroeder-Bernstein Theorem and excluded middle, but "only" about the Schroeder-Bernstein Theorem, which would be ~isbth. Therefore, "This is Metamath 100 proof #25. " should be moved to the comment of ~isbth.

Reply:
I'm not willing to give credit for isbth - that would reduce iset.mm to just ZFC with odd notation. No, in order to count as a solution we need to either prove Schoeder-Bernstein from the IZF axioms (example: https://us.metamath.org/ileuni/bezout.html , another Metamath 100 problem, is stated as in set.mm but needs a very different proof) or show that it is not possible. Of course, for Schoeder-Bernstein it is the latter and for this reason the https://us.metamath.org/ileuni/exmidsbthr.html direction is the more interesting of the two.

A more subtle example is "the square root of two is irrational" where constructive mathematicians have a definition of "irrational" which I think is how we should interpret this problem, but Schroeder-Bernstein doesn't have that sort of issue.

As for where to say this, maybe in mm_100.html? I'll push up a change to make this say "The intuitionistic logic explorer (iset.mm) database shows that Schroeder-Bernstein is equivalent to excluded middle, resolving
this problem in the negative, at exmidsbth"